### PR TITLE
TDR-3055 Use disply property name in download csv.

### PR DESCRIPTION
**Pull request by MancunianSam**
This is using the Name attribute from DisplayProperties.
It still filters on AllowExport in the custom metadata but also checks
if it's active in DisplayProperties as well.

### CODE REVIEW

**Review by vimleshtna — COMMENTED** [review]
Looks good, just a minor comment. Thanks

File: app/controllers/DownloadMetadataController.scala [hunk 1]
```diff
@@ -40,19 +41,24 @@ class DownloadMetadataController @Inject() (
...
+      val filteredMetadata: List[CustomMetadata] = customMetadata.filter(cm => nameMap.keySet.contains(cm.name) && cm.allowExport).sortBy(_.exportOrdinal.getOrElse(Int.MaxValue))
+      val header: List[String] = filteredMetadata.map(f => nameMap.getOrElse(f.name, f.name))
       val fileMetadataRows: List[List[String]] = metadata.files.map { file =>
         val groupedMetadata: Map[String, String] = file.fileMetadata.groupBy(_.name).view.mapValues(_.map(_.value).mkString("|")).toMap
         filteredMetadata.map { customMetadata =>
           groupedMetadata
             .get(customMetadata.name)
             .map { fileMetadataValue =>
-              if (filteredMetadata.find(_.name == customMetadata.name).exists(_.dataType == DataType.DateTime)) {
+              val dataType: Option[DataType] = filteredMetadata.find(_.name == customMetadata.name).map(_.dataType)
```
[the remark this answers is not in the document]
Comment (MancunianSam) [reply app/controllers/DownloadMetadataController.scala]: That's a lot neater, thanks. I've changed it.

**Comment by TomJKing** [comment]
@MancunianSam Could you make sure the e2e tests are updated in line with new column names.

**Review by vimleshtna — APPROVED** [review]
Looks good, thanks

**Pull request merged**